src/_guides/language/effective-dart/documentation.md

```diff
@@ -421,7 +421,7 @@ a flavor of what's supported:
 ///
 /// Code blocks are fenced in triple backticks:
 ///
-/// ```
+/// ```dart
```

Comment:
Looks the build will fail until we update the included file, docs_good.dart

Reply:
Thanks. I always forget to include changes in both places...